helm/name-lookup/templates/solr-deployment.yaml

```diff
@@ -69,6 +69,18 @@ spec:
           volumeMounts:
             - mountPath: /var/solr/data
               name: {{ include "name-lookup.fullname" . }}-solr-data-vol
+      {{- with .Values.app.nodeSelector }}
```

Comment:
I have one comment / suggestion here, since using one generic selector might mean that the whole thing goes to that group of node, i am wondering if big memory components need special values for Node selection , to be able to put them in designated big memory nodes, if so maybe we can define those values as 
``` .Values.solr.nodeSelector``` maybe?

Reply:
I've tried to fix this in 5bc69bb435a3965 -- let me know if you'd like me to make any other changes!